[Bug] bsp/cvitek: see more build warnings

### RT-Thread Version

master

### Hardware Type/Architectures

bsp/cvitek

### Develop Toolchain

Other

### Describe the bug

Introduced by commit 6cbb2c3ee5

when build c906_little:

```shell
CC build/board/cache.o
board/cache.c: In function 'rt_hw_cpu_dcache_ops':
board/cache.c:36:28: warning: passing argument 1 of 'flush_dcache_range' makes integer from pointer without a cast [-Wint-conversion]
   36 |         flush_dcache_range(addr, size);
      |                            ^~~~
      |                            |
      |                            void *
board/cache.c:27:42: note: expected 'uintptr_t' {aka 'long unsigned int'} but argument is of type 'void *'
   27 | inline void flush_dcache_range(uintptr_t start, size_t size) {
      |                                ~~~~~~~~~~^~~~~
board/cache.c:39:26: warning: passing argument 1 of 'inv_dcache_range' makes integer from pointer without a cast [-Wint-conversion]
   39 |         inv_dcache_range(addr, size);
      |                          ^~~~
      |                          |
      |                          void *
board/cache.c:23:40: note: expected 'uintptr_t' {aka 'long unsigned int'} but argument is of type 'void *'
   23 | inline void inv_dcache_range(uintptr_t start, size_t size) {
      |                              ~~~~~~~~~~^~~~~
board/cache.c: In function 'rt_hw_cpu_icache_ops':
board/cache.c:65:26: warning: passing argument 1 of 'inv_icache_range' makes integer from pointer without a cast [-Wint-conversion]
   65 |         inv_icache_range(addr, size);
      |                          ^~~~
      |                          |
      |                          void *
board/cache.c:56:40: note: expected 'uintptr_t' {aka 'long unsigned int'} but argument is of type 'void *'
   56 | inline void inv_icache_range(uintptr_t start, size_t size) {
      |                              ~~~~~~~~~~^~~~~
```

### Other additional context

_No response_

--- a/bsp/cvitek/c906_little/board/cache.c
+++ b/bsp/cvitek/c906_little/board/cache.c
@@ -5,6 +5,7 @@
  *
  * Change Logs:
  * Date           Author       Notes
+ * 2025/01/16     zdtyuiop4444 fix type cast warning
  * 2024/11/26     zdtyuiop4444 The first version
  */
 
@@ -33,10 +34,10 @@ inline void rt_hw_cpu_dcache_ops(int ops, void* addr, int size)
     switch (ops)
     {
     case RT_HW_CACHE_FLUSH:
-        flush_dcache_range(addr, size);
+        flush_dcache_range((uintptr_t)addr, size);
         break;
     case RT_HW_CACHE_INVALIDATE:
-        inv_dcache_range(addr, size);
+        inv_dcache_range((uintptr_t)addr, size);
         break;
     default:
         break;
@@ -62,7 +63,7 @@ inline void rt_hw_cpu_icache_ops(int ops, void* addr, int size)
     switch (ops)
     {
     case RT_HW_CACHE_INVALIDATE:
-        inv_icache_range(addr, size);
+        inv_icache_range((uintptr_t)addr, size);
         break;
     default:
         break;

